index.html

```diff
@@ -431,5 +445,6 @@ <h2 class="heading">Contact me</h2>
         </ul>
       </form>
     </section>
+    <script defer src="main.js"></script>
   </body>
```

Comment:
- [ ] Hi you are calling for a JS file that doesn't exist in your project and the only js file (`navigation.js`) is not called anywhere.

Reply:
Thanks noted.